cdrom/aulas/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.urls import path
-from aulas.views import AulaDetailView, AulaListView, AulaCreateView
+from aulas.views import AulaDetailView, AulaListView, AulaCreateView, AulaUpdateView
 
 
 urlpatterns = [
     path('', AulaListView.as_view(), name='aula_list'),
     path('<int:pk>/', AulaDetailView.as_view(), name='aula_detail'),
     path('aula/add/', AulaCreateView.as_view(), name='aula_add'),
+    path('aulas/<int:pk>/editar/', AulaUpdateView.as_view(), name='aula_edit'),
```

Comment:
```suggestion
    path('lista/', AulaListView.as_view(), name='aula_list'),
    path('<int:pk>/', AulaDetailView.as_view(), name='aula_detail'),
    path('add/', AulaCreateView.as_view(), name='aula_add'),
    path('<int:pk>/editar/', AulaUpdateView.as_view(), name='aula_edit'),
```
Ya estamos dentro de la url `aulas/`